proto/wg/cosmo/node/v1/node.proto

```diff
@@ -195,10 +195,18 @@ message MTLSConfiguration {
   bool insecureSkipVerify = 3;
 }
 
+enum GraphQLSubscriptionProtocol {
+  GRAPHQL_SUBSCRIPTION_PROTOCOL_WS = 0;
+  GRAPHQL_SUBSCRIPTION_PROTOCOL_SSE = 1;
+  GRAPHQL_SUBSCRIPTION_PROTOCOL_SSE_POST = 2;
+}
+
 message GraphQLSubscriptionConfiguration {
   bool enabled = 1;
   ConfigurationVariable url = 2;
-  bool useSSE = 3;
+  /* @deprecated - Kept for backwards compatibility when decoding. Use protocol instead. */
+  optional bool useSSE = 3;
+  GraphQLSubscriptionProtocol protocol = 4;
```

Comment:
How will an old config behave on the router? Is it backward compatible? Please test.

Reply:
I've tested it with an static config generated with wgc from npx. It works and uses ws for the subscriptions (which is the default)